ports/stm32/resethandler.s

```diff
@@ -33,6 +33,16 @@
     .type Reset_Handler, %function
 
 Reset_Handler:
+    #if defined(STM32F7) || defined(STM32H7)
+    /* Check for and handle bootloader request */
+    ldr r0, =_heap_end /* The BL request pattern is stored here in machine.bootloader() */
```

Comment:
Would it not be more efficient to use `_estack`, which is already loaded into sp below.  That would save one instruction.